src/DSC.sol

```diff
@@ -13,14 +13,14 @@ contract DSC is ERC20WithFeesUpgradeable, UUPSUpgradeable {
 	}
 
 	function initialize(
+		address _initialOwner,
 		string memory _name,
 		string memory _symbol,
 		address _feeCollectionAddress,
-		address _minterAddress,
-		address _ownerAddress
+		address _minterAddress
 	) public initializer {
 		__ERC20WithFees_init(
-			_ownerAddress,
+			_initialOwner,
```

Comment:
we need to keep the deployer account and can also directly set the new owner, no?

Reply:
we are setting the owner via a parameter in the initialise call, the deployer account can be whatever, it doesn't have any special privileges after initialisation